docs/background/disaster-recovery.md

```diff
@@ -0,0 +1,54 @@
+# Disaster recovery
+
+When you go on to create a new server in {{brand}}, you will probably
+notice an option named **Disaster recovery**. That particular option is
+enabled by default.
+
+![Disaster recovery is enabled by default for new
+servers](assets/disaster-recovery-option-on-by-default.png)
+
+Even if you choose to disable it for a particular server, keep in mind
+that you have the option to enable it at a later time.
+
+![The disaster recovery feature can be manually activated for any
+server](assets/disaster-recovery-option-manual-activation.png)
+
+In the following, we explain what this option does, how it works in the
+background, and why you should consider enabling it.
+
+## What it is
+
+The *Disaster Recovery* feature, DR for short, is managed from the
+{{gui}} only and applies to servers and volumes that use our
+[Ceph](https://ceph.io) backend.
+
+## How it works
+
+As soon as you enable DR for a server or a single volume, then you start
+getting snapshots for the corresponding [*RADOS Block Device*
+(RBD)](https://docs.ceph.com/en/latest/glossary/#term-Ceph-Block-Device)
+image. Those snapshots are created automatically once per day, and you
+always have the snapshots of the last 10 days. Please keep in mind that
+all RBD snapshots are created in the same Ceph cluster and are not
+replicated remotely. If, for any reason, you delete the original volume,
+then all snapshots will also be deleted, and the snapshot creation
+schedule will be canceled immediately.
+
+## Why enable it
+
+Provided snapshots are available, you can restore a server or a single
+volume to any of those snapshots. For instance, you may discover that
+due to faulty application logic or simply a bug, you are now
+experiencing data corruption. Then, one of your options would be to go
+back in time by restoring one of the available snapshots and keep going
+from there.
+
+You should know that the DR feature creates *point-in-time* snapshots on
+the storage level. This is highly advantageous because the process is
+almost instantaneous, with no write-lock on the RBD image. On the other
+hand, the time required to restore a particular image depends on its
+size. During restoration, the volume is automatically detached before
```

Comment:
You'll need to explain how this works for a boot-from-volume server (which cannot detach its root volume).

Reply:
Understood. I provided the above piece of info based on the attached Wiki ("Disaster recovery questions"). Since I do not know what happens behind the scenes with boot-from-volume servers, I will either need some pointers or we will have to ask for an expert's insight here.

Reply:
@fghaas After reading @Jeeaaasus's explanation and trying to restore a boot-from-volume server to a particular snapshot, I realized that no detachment takes place. Instead, the instance shuts off, the restoration process completes, and then you have to manually power it back on. I am updating the original paragraph accordingly.